scripts/stash.sh

```
@@ -0,0 +1,372 @@
+    if ! list_stashes; then
+        return 1
+    fi
+
+    if [ ${#stashes_refs[@]} -gt 9 ]; then
+        echo "00. Exit"
+    else
+        echo "0. Exit"
+    fi
+    echo
+
+    read_prefix="Select stash number: "
```

Shellcheck (0.10.0)

[warning] read_prefix appears unused. Verify use (or export if used externally).

(SC2034)